test/parallel/test-util-emit-experimental-warning.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+// Flags: --expose-internals
+const common = require('../common');
+const assert = require('assert');
+
+// This test ensures that the emitExperimentalWarning in internal/util emits a
+// warning when passed a unsupported feature and that it simply returns when
+// passed a supported feature.
+
+const { emitExperimentalWarning } = require('internal/util');
+
+process.on('warning', common.mustCall((warning) => {
+  assert(/is an experimental feature/.test(warning.message));
+}, 2));
```

Comment:
Can you please add an empty line in between the `process.on('warning')` and `emitExperimentalWarning` calls

Reply:
@maclover7  : Thanks for the feedback. I've updated the PR. Kindly review now.